src/components/value-list/value-list.tsx

```diff
@@ -103,30 +103,30 @@ export class ValueList<
   @Prop() selectionFollowsFocus = false;
 
   /**
-   * When "drag-enabled" is true and active, specifies accessible context to the `calcite-value-list-item's` initial position.
+   * When "drag-enabled" is true and active, specifies accessible context to the `calcite-value-list-item`'s initial position.
    *
-   * use ${position} of ${total} as placeholder for indexes.
+   * use ${position} of ${total} as placeholder for displaying indices.
```

Comment:
nitpick: Can we capitalize "**U**se" throughout (lines 108, 115, 122, 129)?

Reply:
sure thing.